examples/3dtiles.html

```diff
@@ -96,7 +96,7 @@
 
             // Create a new Layer 3d-tiles For DiscreteLOD
             // -------------------------------------------
-            var $3dTilesLayerDiscreteLOD = new itowns.GeometryLayer('3d-tiles-discrete-lod', globe.scene);
+            var $3dTilesLayerDiscreteLOD = new itowns.GeometryLayer('3d-tiles-discrete-lod', new itowns.THREE.Group());
```

Comment:
What about having Layer.object3D defaulted to a new THREE.Group [here](https://github.com/iTowns/itowns/pull/539/files#diff-61e012c3e00bdc510be33d9eac684b2aR61) instead of throwing?

Reply:
If we silently create a THREE.Group the user will still need to add it to the scene. And if it's not added to the scene the layer will be invisible.

So I'd prefer throw here to make sure the user is aware that displaying the layer relies on a three object.

Reply:
> If we silently create a THREE.Group the user will still need to add it to the scene

Nope, because we add it [here](https://github.com/iTowns/itowns/pull/539/files#diff-d297dcf7b9d0e696ece4b1f6e1d7c69dR313) since this patch. It seems reasonable to have a `THREE.Group` by default to me, as you nearly always do:

```js
pointcloud = new itowns.GeometryLayer('pointcloud', new itowns.THREE.Group());
```
It'll just save the extra argument. (not a big deal though)